modules/service/src/main/scala/lucuma/odb/graphql/enums/Enums.scala

```diff
@@ -122,15 +126,27 @@ final class Enums(
 }
 
 object Enums {
+  case class ScalaEnum(
+    postgresName: String,
+    enumType: EnumType
+  )
+
+  // enums defined in scala for which we want to validate the postgres enums and add to the schema
+  val scalaEnums: List[ScalaEnum] =
+    List(
+      ScalaEnum("e_too_activation", Enumerated[ToOActivation].toEnumType("ToOActivation", "Target of Opportunity Activation")(_.label))
```

Comment:
The description string ("Target of Opportunity Activiation") doesn't seem to be actually be used for anything?

Reply:
However, we can add comments before the `DUMMY` entries in OdbSchema.graphql and they will be retained in the final schema.

Reply:
What happens if the validation with postgres fails?

Reply:
The service doesn't start. Same thing as if there is a problem loading the dynamic enums.